### DISCUSSION TITLE
Does pg-boss have a Python implementation?

### DISCUSSION THREAD

**Discussion by roy-moven** [timgit/pg-boss#408] [Q&A] [answered]
I'm considering replacing some Kafka queues with pg-boss. Most of my code is in NodeJS but I have a few components that consume from the Kafka queues that are implemented in Python. Is there a Python implementation of pg-boss that is compatible with the NodeJS implementation, such that jobs created in NodeJS could be consumed in Python and vice versa?

**Comment by timgit** [comment, marked as answer]
Unless you need specific features like denouncing or uniqueness, you can directly insert jobs from python using SQL. Consuming jobs from Python is going to be much more difficult since the fetch command is more complicated than insert.
